File: Java/프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.stream.IntStream;
class Solution {
    public int[] solution(int[] numbers) {
        int[] answer = {};
        ArrayList<Integer> set = new ArrayList<>();
        for(int i=0;i<numbers.length;i++){
            for(int j= i+1; j<numbers.length;j++){
                set.add(numbers[i]+numbers[j]);
            }
        }
        answer = set.stream().distinct().mapToInt(Integer::intValue).toArray();
        Arrays.sort(answer);
        return answer;
    }
}

File: Kotlin/프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.kt
class Solution {
    fun solution(numbers: IntArray): IntArray {
        var answer = mutableSetOf<Int>()
        for(i in 0..numbers.lastIndex) {
            if(i<numbers.lastIndex) {
                for(j in (i+1)..numbers.lastIndex) {
                    answer.add(numbers[i]+numbers[j])
                }
            }  
        }
        
        return answer.sorted().toIntArray()
    }
}